code/game/machinery/computer/card.dm

```diff
@@ -6,32 +8,119 @@
 	icon_keyboard = "id_key"
 	icon_screen = "id"
 	req_access = list(access_change_ids)
-	circuit = "/obj/item/weapon/circuitboard/card"
+	circuit = /obj/item/weapon/circuitboard/card
+	light_color = LIGHT_COLOR_LIGHTBLUE
 	var/obj/item/weapon/card/id/scan = null
 	var/obj/item/weapon/card/id/modify = null
 	var/mode = 0.0
 	var/printing = null
+	
+	//Cooldown for closing positions in seconds
+	//if set to -1: No cooldown... probably a bad idea
+	//if set to 0: Not able to close "original" positions. You can only close positions that you have opened before
+	var/change_position_cooldown = 60
+	//Jobs you cannot open new positions for
+	var/list/blacklisted = list(
+		"AI",
+		"Cyborg",
+		"Captain",
+		"Head of Personnel",
+		"Head of Security",
+		"Chief Engineer",
+		"Research Director",
+		"Chief Medical Officer",
+		"Magistrate",
+		"Blueshield",
+		"Nanotrasen Representative",
+		"Security Pod Pilot",
+		"Brig Physician",
+		"Mechanic",
+		"Barber",
+		"Civilian",	
+		"Chaplain"
+	)
```

Comment:
Wouldn't it be more typo-proof to use ```datum/job``` types and compare strictly by type in format_job_slots instead of name?
The main issue I can see is that it would greatly complicate admin-VVing of the list during rounds if they for any reason would want to add/remove blacklisted jobs.